__tests__/posts.ts

```diff
@@ -2003,6 +2038,31 @@ describe('mutation viewPost', () => {
   });
 });
 
+describe('trigger increment_squad_views_count', () => {
```

Comment:
We have triggers in separate test file `__tests__/triggers/post.ts`, sorry for missing this in earlier review.

Reply:
Ah, sure, we can move it.